projects/control-service/projects/helm_charts/pipelines-control-service/values.yaml

```diff
@@ -820,7 +820,7 @@ customStartupProbeProbe: {}
 datajobTemplate:
   enabled: false
   template:
-    apiVersion: batch/v1beta1
+    apiVersion: batch/v1
```

Comment:
what are then the supported values after the change?

Reply:
v1 is only supported value.

Reply:
nit: since the only value, then why at all configurable?

Reply:
Its not configurable, its like the way it says cronJob below. 
Its just describing the values for the template.

Reply:
nit: so when you have a configuration with 1 possible option, then it is not configurable. why not removing the configuration entirely, since it cannot be anything else than this particular value..?

Reply:
This isn't configuration. 
It is the template for a cronJob. 

For example you wouldn't suggest removing the line  `kind: CronJob` two lines below it would you? 

We need to specify it here so k8s knows what apiVersion we are using.
Does that answer your q?

Reply:
oh yes, that's the default cronjob `apiVersion` value, used by users unless reconfigured.
This is definitely an API change, that requires consideration and documentation accordingly.
Currently, Kubernetes instance is provisioned externally as a prerequisite to installing VDK.